FEM/invariants.h

```diff
@@ -0,0 +1,131 @@
+#ifndef INVARIANTS_H
+#define INVARIANTS_H
+
+#include <cfloat>
+// FEM-Makros
+#include "makros.h"
+#include "tools.h"
+
+#include "Eigen/Dense"
+
+namespace SolidMath
+{
+
+static Eigen::Matrix<double,6,6> P_dev; //deviatoric projection matrix
```

Comment:
In general, global variables are not recommended in c++. isn't it possible to let Process or Assembler class own these and pas them to `InitialiseProjectionTensors()`?